presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -875,4 +886,138 @@ public HiveClientConfig setUseParquetColumnNames(boolean useParquetColumnNames)
         this.useParquetColumnNames = useParquetColumnNames;
         return this;
     }
+
+    public HiveMetastoreAuthenticationType getHiveMetastoreAuthenticationType()
+    {
+        return hiveMetastoreAuthenticationType;
+    }
+
+    public enum HiveMetastoreAuthenticationType
+    {
+        SIMPLE,
+        SASL
+    }
+
+    @Config("hive.metastore.authentication.type")
+    @ConfigDescription("Hive Metastore authentication type. Possible values are: SIMPLE, SASL. Defaults to: SIMPLE")
```

Comment:
@electrum do we normally format like this?  IIRC the start up message includes the default value, so I don't think we need that.